db/seeds.rb

```diff
@@ -1,5 +1,6 @@
 Patient.destroy_all
 User.destroy_all
+Clinic.destroy_all
```

Comment:
IIRC Clinic data is embedded in Patient, so we don't need to specifically destroy, right?